### ISSUE TITLE
undefined symbol in icc 14.0.1-linux64 build

### ISSUE THREAD

**Issue by jiahao** [JuliaLang/openspecfun#30]
Building julia release-0.3 with JuliaLang/julia#10181 results in the following error:

```
julia $ make test
...
        From worker 6:       * math
/home/jiahao/julia-intel/usr/bin/./julia: symbol lookup error: /home/jiahao/julia-intel/usr/bin/../lib/libopenspecfun.so: undefined symbol: complex_
Worker 6 terminated.
```

**Comment by tkelman** [comment]
See #38 - I think we need to bump the submodule in Julia to include those changes. Probably best to test the CI waters via a PR in case anything breaks...

**Comment by tkelman** [comment]
28 rather

**Comment by jiahao** [comment]
Closed by #10198 #10200
